src/Form/FieldSelect.js

```diff
@@ -73,6 +73,7 @@ class FieldSelect extends React.Component {
       <Dropdown {...classes}
         initialID={startValue.id}
         items={dropdownItems}
+        matchButtonWidth={matchButtonWidth}
```

Comment:
This is done now. We just need to be sure not to do this for actual DOM elements, because React 15 will throw a warning. https://facebook.github.io/react/warnings/unknown-prop.html